server/re/src/printMS.cpp

```diff
@@ -171,12 +171,16 @@ int writeBytesBuf( msParam_t* where, msParam_t* inBuf, ruleExecInfo_t *rei ) {
     }
 
     if ( inBuf->inpOutBuf ) {
+        const auto writeStr_len{inBuf->inpOutBuf->len + 1};
         /* Buffer might no be null-terminated */
-        writeStr = ( char* )malloc( inBuf->inpOutBuf->len + 1 );
+        writeStr = (char*) malloc(writeStr_len);
         if (nullptr != inBuf->inpOutBuf->buf) {
             strncpy(writeStr, (char*) inBuf->inpOutBuf->buf, inBuf->inpOutBuf->len);
```

Comment:
Let's go ahead and give these the `static_cast` treatment while we're in here.
```suggestion
        writeStr = static_cast<char*>(malloc(writeStr_len));
        if (nullptr != inBuf->inpOutBuf->buf) {
            strncpy(writeStr, static_cast<char*>(inBuf->inpOutBuf->buf), inBuf->inpOutBuf->len);
```

Reply:
Sounds good

Reply:
Done.